src/main/java/org/jenkinsci/plugins/github/extension/GHSubscriberEvent.java

```diff
@@ -0,0 +1,39 @@
+package org.jenkinsci.plugins.github.extension;
+
+import edu.umd.cs.findbugs.annotations.CheckForNull;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import javax.servlet.http.HttpServletRequest;
+import jenkins.scm.api.SCMEvent;
+import org.kohsuke.github.GHEvent;
+
+/**
+ * An event for a {@link GHEventsSubscriber}.
+ *
+ * @since 1.25.2
+ */
+public class GHSubscriberEvent extends SCMEvent<String> {
```

Comment:
So you are adding hard dependency on plugin that community never discussed to design?